src/Microsoft.SqlTools.SqlCore/Microsoft.SqlTools.SqlCore.csproj

```diff
@@ -36,4 +42,23 @@
 		<InternalsVisibleTo Include="Microsoft.SqlTools.ServiceLayer.Test.Common" />
 		<InternalsVisibleTo Include="DynamicProxyGenAssembly2" />
 	</ItemGroup>
+
+	<PropertyGroup>
+		<TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
```

Comment:
What is all this for? What do you need to filter, and could you not use a nuspec instead?